api.md

```diff
@@ -412,6 +413,39 @@ queries.
 
 ---
 
+## CREATE INDEX [](create_index)
```

Comment:
maybe the heading should be `CREATE INDEX ... WITH (timescaledb.transaction_per_chunk)`?